src/app/hooks/use-validation.ts

```diff
@@ -44,7 +46,10 @@ export const useValidation = () => {
 			usernameRegistration: usernameRegistration(t),
 			validatorRegistration: validatorRegistration(t),
 			verifyMessage: verifyMessage(t),
+			...(properties?.network == undefined ? {}: {
+				delegateRegistration: delegateRegistration(t, properties?.network)
+			})
```

Comment:
@goga-m True, it will throw an error if no network is passed, but I made sure to pass the network everywhere the validator is used. Regardless, I made an adjustment to return an alternative method that throws an exception in case the method is used without the network.

Let me know what do you think -> https://github.com/ArdentHQ/arkvault/pull/440/commits/256d0f95dc2de6c0e659ee722a2e92375ce8e6cc

Reply:
added a card for that here https://app.clickup.com/t/86dt8g7cu @goga-m @alfonsobries to not have this PR be blocked by it